pkg/web/webtest/webtest.go

```diff
@@ -129,7 +127,9 @@ func requestContextFromRequest(req *http.Request) *models.ReqContext {
 }
 
 func requestContextMiddleware() web.Handler {
-	return func(res http.ResponseWriter, req *http.Request, c *models.ReqContext) {
+	return func(res http.ResponseWriter, req *http.Request) {
+		c := ctxkey.Get(req.Context()).(*models.ReqContext)
```

Comment:
Sure it is safe not to check "ok" on the type assertion?

Reply:
I believe so. These are framework internals, and if the framework ever manages to inject something else than `*models.ReqContext` into this spot, it is definitely doing wrong and I think I like a loud and unmissable panic in this case